lucene/benchmark/conf/indexing-flush-by-RAM-multithreaded.alg

```diff
@@ -53,7 +53,8 @@ log.queries=true
 
     { "Populate"
         CreateIndex
-        [{ "MAddDocs" AddDoc } : 5000] : 4
+        #[{ "MAddDocs" AddDoc } : 5000] : 4
+        [{ {{"MAddDocs" AddDoc } : 5000} FlushIndex }  ] : 20
```

Comment:
Maybe 1) remove the old (commented out) line entirely?  And 2), 20 threads seems a bit much for a "typical" box now?  Maybe go to 8 instead?

Hmm, it would be nice if benchmarks made the local machine's CPU concurrency available (`Runtime.getRuntime().availableProcessors()`) so we could set this dynamically.  But that is for another day!

Reply:
1) Sure Mike, we will remove the old (commented out) line entirely.
2) We can change #parallel indexing thread to 8, to work with a typical box.